### DISCUSSION TITLE
A way to handle pseudo classes in Tailwindcss

### DISCUSSION THREAD

**Discussion by zbandhan** [tailwindlabs/tailwindcss#10403] [Ideas]
First of all, I like to thank the Tailwindcss team for giving us such a nice way of writing CSS without visually writing it. 

**I found pseudo classes need to be written multiple times for adding multiple classes with it.**

For example: `before:absolute before:w-full before:h-full`

Don't know whether it's possible to implement but my guess is that the following writing style might be convenient and could occupy  less space in DOM.

like
```
before:{absolute w-full h-[95px]}
md:{w-1/2 bg-blue-500}

```

**Comment by wongjn** [comment]
[Here's a response to a similar suggestion](https://github.com/tailwindlabs/tailwindcss/discussions/9053#discussioncomment-3350470):

> We get this question so many times, so I would like to redirect you here for more information: https://twitter.com/adamwathan/status/1461519820411789314

**Comment by zbandhan** [comment]
This is something WindiCSS has already been implemented.
  
![image](https://user-images.githubusercontent.com/10957285/214143279-2d9289c9-0e67-4750-b7d8-867882fd98f8.png)

**Comment by Charleswilkins** [comment]
Using utilities to style elements on hover, focus, and more. Every utility class in Tailwind can be applied conditionally by adding a modifier to the beginning of the class name that describes the condition you want to target.  
[Caregiver Connect](https://www.caregiver-connect.org/)
